dispatcher/subscription.go

```diff
@@ -4,10 +4,10 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/google/uuid"
```

Comment:
should also sort imports on test file
same for:
- hub/export_test.go
- mocks/consumerMock.go

Reply:
done